src/api/layers/AggregateLayer.class.js

```diff
@@ -48,6 +50,8 @@ export default class AggregateLayer extends GeoAdminLayer {
         name,
         id,
         opacity,
+        attribution,
```

Comment:
maybe `attributionName` to make it more clear (i.e. attribution consists of attributionName and attributionUrl)